### ISSUE TITLE
You Comments about the Paper

### ISSUE THREAD

**Issue by royshan** [Nickkontschak/Replication-The-virtue-of-complexity-in-machine-learning-portfolios#2]
Hi, Nkonts.

Thank you for sharing your replication of the paper. I wonder what are you final comments/conclusion about the paper? Have you found any issues or flaws?

Again, thank you!

**Comment by eweisbrod** [comment]
Yes, I also found your site helpful. Are you in academia? Do you have a webpage or CV?

**Comment by royshan** [comment]
I found good criticism from Prof. Berk's Comment on the paper. But I think the over-parameterization strategy is still worth trying.
